src/pykeen/experiments/cli.py

```diff
@@ -194,19 +193,16 @@ def ablation(
 ) -> None:
     """Generate a set of HPO configurations.
 
-    A sample file can be run with ``pykeen experiment ablation tests/resources/hpo_complex_nations.json``.
+    A sample file can be run with ``pykeen experiments ablation tests/resources/hpo_complex_nations.json``.
     """
-    from pykeen.ablation import ablation_pipeline
+    from ..ablation.ablation import prepare_ablation_from_path, _create_path_with_id, _run_ablation_experiments
+    directory = _create_path_with_id(directory=directory)
```

Comment:
why is the CLI implementing logic? shouldn't there be a function that takes the JSON and just does everything?

Reply:
Done in deac886fdcd991c89ddbe7a84e42f0c90123c07d.